jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants.properties

```diff
@@ -68,39 +68,39 @@ DDValidationNotValidResolver=Not valid resolver selected for {0}
 DDValidationUnexpectedError=Unexpected error while validating deployment descriptor: {0}
 CMMNFileTypeDescription=Case Management Model
 CMMNFileTypeShortName=CMMN
-ServiceTasksRepositoryPreferences.Label=Service Tasks Repository Preferences
+ServiceTasksRepositoryPreferences.Label=Custom Tasks Repository Preferences
 MavenInstall=Install as Maven artifact
-MavenInstallHelp=Instructs if enabled service tasks should be installed into Maven repository
-InstallPomDeps=Install service task dependencies into project
-InstallPomDepsHelp=Instructs that service task dependencies are added as project dependencies upon installation
-UseVersionRange=Use version range when installing service task into a project
-UseVersionRangeHelp=Instructs that a version range will be used when installing service task in projects
+MavenInstallHelp=Instructs if enabled custom tasks should be installed into Maven repository
+InstallPomDeps=Install custom task dependencies into project
+InstallPomDepsHelp=Instructs that custom task dependencies are added as project dependencies upon installation
+UseVersionRange=Use version range when installing custom task into a project
+UseVersionRangeHelp=Instructs that a version range will be used when installing custom task in projects
 ServiceTaskList=Custom Tasks Administration
-NoServiceTasksFound=No service tasks found
+NoServiceTasksFound=No custom tasks found
 BaseSettings=Settings
-ServiceTasks=Service Tasks
```

Comment:
Just a really minor comment, not just in this line, but in general we're changing only the literal "Service" to "Custom" in the "values" for the properties file, at some point would also make sense to change the "keys", so once just coding on top of this, the developer will not be confused about service vs custom. 
Anyway, from the user perspective, it works, so we can go ahead.